plugins/time-to-read-block/package.json

```
@@ -0,0 +1,30 @@
+{
+	"name": "time-to-read-block",
+	"version": "1.0.0",
+	"description": "Block to display the minutes required to finish reading a post.",
+	"author": "WordPress.org",
+	"license": "GPL-2.0-or-later",
+	"main": "build/index.js",
+	"scripts": {
+		"build": "wp-scripts build",
+		"format": "wp-scripts format",
+		"lint:css": "wp-scripts lint-style",
+		"lint:js": "wp-scripts lint-js",
+		"packages-update": "wp-scripts packages-update",
+		"plugin-zip": "wp-scripts plugin-zip",
+		"start": "wp-scripts start"
+	},
+	"devDependencies": {
+		"@wordpress/scripts": "27.7.0"
+	},
+	"dependencies": {
+		"@wordpress/block-editor": "^12.24.0",
```

I think we should reference the local packages that are present in the project, like in other places, example:

https://github.com/WordPress/gutenberg/blob/4d1c083f011a4435c5f12df8d1c14a333df1758c/packages/block-library/package.json#L33-L66

Here, the path will be a bit different:

```suggestion
		"@wordpress/block-editor": "file:../../packages/block-editor",
```

---

Yes, that seemed to work as expected. Also done in https://github.com/WordPress/gutenberg/pull/61504/commits/30cab7a38d9e4e9f23c89a6eec4e45a5da348ea3.

---

@gziolo Thinking through this a bit more, I think we would need to specify the latest version of packages shipped in WordPress, to ensure compatibility.

Block plugins should work without Gutenberg active. Since `@wordpress` packages are transformed to reference `window.wp` as part of the js build process, I believe that production block code could end up using older package versions and unexpectedly break if newer APIs were not available in those older versions.

Does that seem accurate? What do you think?

---

Then again, it doesn't really seem to matter what package versions are specified here, as either way the actual code in the browser is going to use whatever `window.wp` provides, independent of the packages here in the repo.

The important thing will be running e2e tests without Gutenberg active to ensure the block plugin works with the latest WordPress release even when the Gutenberg plugin is disabled.